docs/how-to-use-paperclip-with-minio.md

```diff
@@ -0,0 +1,37 @@
+# How to use Paperclip with Minio Server [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Paperclip is intended as an easy file attachment library for ActiveRecord. In this recipe we will learn how to configure  Minio as Object Storage backend for Paperclip.
+
+Minio can be used to upload contents like images, music and docs with Paperclip as an object storage backend.
+
+## 1. Prerequisites
+
```

Comment:
Minio Server is installed and running. Please follow Minio Quickstart guide <link> to install. 

This recipe uses https://play.minio.io:9000. Play is a hosted Minio server for testing and development purpose only.